core/templates/services/playthrough-issues-backend-api.service.spec.ts

```diff
@@ -163,51 +147,44 @@ describe('PlaythroughIssuesBackendApiService', () => {
       let playthroughIssue = playthroughIssueObjectFactory
         .createFromBackendDict(backendIssues[0]);
 
-      playthroughIssuesBackendApiService.fetchIssues('7', 1).then(
-        () => {
-          playthroughIssuesBackendApiService.resolveIssue(
-            playthroughIssue, explorationId, 1).then(
-            successHandler, failureHandler);
-        });
+      playthroughIssuesBackendApiService.fetchIssues('7', 1)
+        .then(
+          () => playthroughIssuesBackendApiService.resolveIssue(
+            playthroughIssue, explorationId, 1))
+        .then(successHandler, failureHandler);
       let req = httpTestingController.expectOne(
         '/issuesdatahandler/7?exp_version=1');
-      expect(req.request.method).toEqual('GET');
       req.flush(backendIssues);
       flushMicrotasks();
 
       req = httpTestingController.expectOne(
-        '/resolveissuehandler/7'
-      );
-      expect(req.request.method).toEqual('POST');
+        '/resolveissuehandler/7');
       req.flush(backendIssues);
       flushMicrotasks();
 
-      // resolveIssue does not return any resolve promise method.
+      // resolveIssue does not return a value.
       expect(successHandler).toHaveBeenCalled();
       expect(failureHandler).not.toHaveBeenCalled();
     }));
 
     it('should use the rejection handler when try to get non fetched issue',
       fakeAsync(() => {
+        let successHandler = jasmine.createSpy('success');
+        let failHandler = jasmine.createSpy('fail');
         let explorationId = '7';
         let playthroughIssue = playthroughIssueObjectFactory
           .createFromBackendDict(backendIssues[0]);
 
-        let successHandler = jasmine.createSpy('success');
-        let failHandler = jasmine.createSpy('fail');
         playthroughIssuesBackendApiService.resolveIssue(
           playthroughIssue, explorationId, 1).then(successHandler, failHandler);
         let req = httpTestingController.expectOne(
-          '/resolveissuehandler/' + explorationId
-        );
-        expect(req.request.method).toEqual('POST');
```

Comment:
Why is this being removed -- it's good to check the request method too?

Reply:
Hmm I kind of disagree, I don't think it actually matters whether the request was GET/POST/etc., that's really an implementation detail that doesn't change any observable behavior. WDYT?

Reply:
I've seen this as a common pattern over web that testing HTTP requests involves testing of the method too. For example this blog: https://medium.com/better-programming/testing-http-requests-in-angular-with-httpclienttestingmodule-3880ceac74cf

Reply:
Hmm I wasn't convinced entirely through that link (since it just says it can be done, not that it should be done), but after looking up best practices for making HTTP requests: https://blog.angular-university.io/angular-http/. I can see that expecting GET/POST through this lens is useful, maybe we should update our best practices/angular migration guide to match this view?

Reply:
I've started an email thread for further discussion, PTAL when you can!